labgrid/remote/client.py

```diff
@@ -894,6 +894,28 @@ def tmc_channel(self):
         for k, v in sorted(data.items()):
             print("{:<16s} {:<10s}".format(k, str(v)))
 
+    def write_image(self):
+        place = self.get_acquired_place()
+        target = self._get_target(place)
+        drv = None
+        from ..resource.remote import NetworkUSBMassStorage, NetworkUSBSDMuxDevice
+        from ..driver import NetworkUSBStorageDriver
+        try:
+            drv = target.get_driver(NetworkUSBStorageDriver)
+        except NoDriverFoundError:
+            for resource in target.resources:
+                if isinstance(resource, NetworkUSBSDMuxDevice, NetworkUSBMassStorage):
```

Comment:
`TypeError: isinstance expected 2 arguments, got 3`
This should be:
`if isinstance(resource, (NetworkUSBMassStorage, NetworkUSBSDMuxDevice)):`